src/TextField.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import cx from 'classnames';
+import Label from './Label';
+
+/**
+ * @module TextField
+ */
+class TextField extends React.Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			value: props.value,
+			hasError: props.error,
+			error: props.error
+		};
+		this.handleChange = this.handleChange.bind(this);
+	}
+
+	handleChange(e) {
+		this.setState({ value: e.target.value });
+	}
+
+	render() {
+		const {
+			inputId,
+			name,
+			label,
+			labelClassName,
+			placeholder,
+			formAttrs,
+			className,
+			children,
+			...other
```

Comment:
I'm thinking name should be a required attribute. Is that possible to make it required using `other`?